Xamarin.Forms.Core/EasingTypeConverter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using static Xamarin.Forms.Easing;
+
+namespace Xamarin.Forms
+{
+	[Xaml.ProvideCompiled("Xamarin.Forms.Core.XamlC.EasingTypeConverter")]
+	[Xaml.TypeConversion(typeof(Easing))]
+	public class EasingTypeConverter : TypeConverter
+	{
+		public override object ConvertFromInvariantString(string value)
+		{
+			if (string.IsNullOrWhiteSpace(value))
+				return null;
+
+			value = value?.Trim() ?? "";
+			var parts = value.Split('.');
+			if (parts.Length == 2 && parts[0] == nameof(Easing))
+				value = parts[parts.Length - 1];
+
+			switch (value)
+			{
+				case string easing when AreEqual(easing, nameof(Linear)): return Linear;
```

Comment:
there's no need to try casting the value to string each time. at this time, we're already sure it's a non-empty string

Reply:
I did it for case-insensitive strings comparison.
I don't know how to simplify this syntax( Appreciate any hints

Reply:
the most efficient way to write this (if you care about perf) is as follow.

```csharp
if (value.Equals(name(SinIn))
   return SinIn;
if (value.Equals(name(SinOut))
   return SinOut;
if (value.Equals(name(SinInOut))
   return SinInOut;
...
```

- going through the helper method reduce the number of method invocation
- there's no locals created for the method (from the pattern matching)